operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -475,91 +481,107 @@ private void logThrowable(Throwable throwable) {
   }
 
   /**
-   * Begin activity to align domain status with domain resource.
-   * @param info domain presence info
-   * @param explicitRecheck if explicit recheck
+   * Begin activity to align the cached domain status with the value reaa from Kubernetes.
+   * @param liveInfo domain presence info read from Kubernetes
+   * @param explicitRecheck true if we are to skip comparing the reported domain to the cached value
    * @param isDeleting if is deleting domain
    * @param isWillInterrupt if will interrupt already running activities
    */
   public void makeRightDomainPresence(
-      DomainPresenceInfo info,
+      DomainPresenceInfo liveInfo,
       boolean explicitRecheck,
       boolean isDeleting,
       boolean isWillInterrupt) {
-    Domain domain = info.getDomain();
-    String ns = info.getNamespace();
-    String domainUid = info.getDomainUid();
-
-    if (delegate.isNamespaceRunning(ns)) {
-      DomainPresenceInfo existing = getExistingDomainPresenceInfo(ns, domainUid);
-      if (existing != null) {
-        Domain current = existing.getDomain();
-        if (current != null) {
-          // Is this an outdated watch event?
-          if (domain != null
-              && KubernetesUtils.isFirstNewer(current.getMetadata(), domain.getMetadata())) {
-            LOGGER.fine(MessageKeys.NOT_STARTING_DOMAINUID_THREAD, domainUid);
-            return;
-          }
+    if (!delegate.isNamespaceRunning(liveInfo.getNamespace())) {
```

Comment:
should we also check for isDeleting?

Reply:
No. `isDeleting` is passed into `internalMakeRightDomainPresence` and handled there. The purpose of the checks in this method is to determine whether to call that method.